README.md

```diff
@@ -63,6 +64,8 @@ podman push quay.io/my-container-registry-namespace/my-manifest-bundle:latest
 
 Of course, this build step can be done with any other OCI spec container tools like `docker`, `buildah`, `libpod`, etc.
 
+Note that you do not create your bundle manually. [Operator-SDK](https://github.com/operator-framework/operator-sdk) provide features and helpers to build, to update, to validate and to test bundles for projects which follows the SDK layout or not. For more information check its documentations over [Integration with OLM](https://sdk.operatorframework.io/docs/olm-integration)
```

Comment:
I would remove the first sentence `Note that you do not create your bundle manually.`
This is an upstream project so we shouldn't dictate what users can or can't do manually on their own. Simply pointing out that SDK provides tooling and features to build, to update, to validate and to test bundles is sufficient.

Reply:
It is missing a word actually : note that you do not need to create the ....